src/main/java/study/StringAddCalculator.java

```diff
@@ -0,0 +1,51 @@
+package study;
+
+import java.util.Arrays;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringAddCalculator {
+
+    private static String MATCH_PATTERN = "//(.)\n(.*)";
+    private static String COMMA_OR_COLON = ",|:";
+
+    // 음수 체크
+    private static boolean isNagative(int number) {
+        return number < 0;
+    }
+
+    // split
+    private static String[] split(String text) {
+        Matcher m = Pattern.compile(MATCH_PATTERN).matcher(text);
+        if (m.find()) {
+            String customDelimiter = m.group(1);
+            return m.group(2).split(customDelimiter);
+        }
+        return text.split(COMMA_OR_COLON);
+    }
+
+    public static int splitAndSum(String number) {
+        if (validate(number)) return 0;
+
+        return sum(split(number));
+    }
+
+    // 빈 문자열 또는 null 체크
+    private static boolean validate(String text) {
+        return text == null || text.isEmpty();
+    }
+
+    // sum
+    private static int sum(String[] numbers) {
+        return Arrays.stream(numbers).mapToInt(StringAddCalculator::getPositiveNumber).sum();
```

Comment:
```suggestion
        return Arrays.stream(numbers)
           .mapToInt(StringAddCalculator::getPositiveNumber)
           .sum();
```
가독성을 위해 한줄에 하나의 점 `.` 을 찍어주세요.

Reply:
한줄에 하나의 점을 찍도록 수정하였습니다.